src/main/webapp/js/userMapData.json

```diff
@@ -425,6 +466,7 @@
     "Rotterdam School of Management",
     "Rotterdam University of Applied Science",
     "Saxion University",
+    "Saxion University of Applied Sciences",
```

Comment:
This is the same as "Saxion University" but since it is the more proper name you can delete that one instead.